lib/matplotlib/font_manager.py

```diff
@@ -154,7 +154,7 @@ def list_fonts(directory, extensions):
     extensions = ["." + ext for ext in extensions]
```

Comment:
Doesn't this line mean that `'.'` *is* being included in the extension?

Reply:
Yes, I missed it, but already spotted and reverted the change.